### DISCUSSION TITLE
尝试查看首页页脚内容时闪烁并拉回上部

### DISCUSSION THREAD

**Discussion by DAilyDAwn** [theme-shoka-x/hexo-theme-shokaX#47] [Q&A] [answered]
查看了issues，discussions及各处博客，没找到这个问题，应该不是bug？刚刚入门hexo，不太清楚哪里造成的
在首页滚动到右下角回到顶部显示70%时会出现一帧正常页脚的配置，如随机文章，最新评论和下面一些文字
![1 (2)](https://user-images.githubusercontent.com/16446943/223209303-342a54fd-1572-47e9-b113-b89d996ff982.gif)
而文章底下却显示比较正常，但尝试添加几个评论后，发现最新评论依然是空的，且重新部署后依旧
![image](https://user-images.githubusercontent.com/16446943/223210436-0a66b297-0a36-48bf-9b57-39bfaef89f55.png)

**Comment by zkz098** [comment, marked as answer]
Shoka时期就存在的老问题，目前还没找到具体原因。
解决方法是多写几篇文章，一般来说3篇左右就能解决

**Reply by zkz098** [reply]
Waline最新评论不显示等我下午排查一下

另外你这个评论是不是处于待审核状态，待审核的评论只有自己可见且不会在最新评论中显示

**Reply by DAilyDAwn** [reply]
上面还测试了几个通过和垃圾状态的评论，我了解了，感谢

**Reply by D-Sketon** [reply]
大概率是sidebar吸顶造成的抽动

**Reply by zkz098** [reply]
> 上面还测试了几个通过和垃圾状态的评论，我了解了，感谢

刚刚我看了一下，你的waline跨域设置可能存在问题
![image](https://user-images.githubusercontent.com/92242020/224007070-193fc0f7-61f9-4b58-82c1-af2d428cfc5a.png)
因为我本人使用的是twikoo，所以对waline这方面设置并不是很了解，也许你可以看看[vercel指南](https://vercel.com/guides/how-to-enable-cors#enabling-cors-using-vercel.json) @DAilyDAwn

**Reply by zkz098** [reply]
> > 上面还测试了几个通过和垃圾状态的评论，我了解了，感谢
> 
> 刚刚我看了一下，你的waline跨域设置可能存在问题 ![image](https://user-images.githubusercontent.com/92242020/224007070-193fc0f7-61f9-4b58-82c1-af2d428cfc5a.png) 因为我本人使用的是twikoo，所以对waline这方面设置并不是很了解，也许你可以看看[vercel指南](https://vercel.com/guides/how-to-enable-cors#enabling-cors-using-vercel.json) @DAilyDAwn

刚又看了一遍，我的锅()
waline的serverURL的存在一些问题，comment的请求是`${serverURL}comment`但rencent的请求是`${serverURL}/comment`
然后ShokaX没有针对这个做特殊处理，导致308就会把fetch炸掉
`v0.2.0-rc6`马上修 (

目前把serverURL的末尾斜杠去掉应该能解决
